gctscript/modules/gct/exchange.go

```diff
@@ -0,0 +1,330 @@
+package gct
```

Comment:
that's correct the reason for calling it exchange is that it interacts with the exchange interface from GCT

GCT is the module name
exchange.go -> deals with exchange related
another one might be 
logging.go that allows you to interact with logging features

the double naming of things that already have a place outside of scripting was also a concern for me like exchange and exchange will try think of a better solution if you have any ideas feel free to let me know :D

Reply:
Yeah I didn't really help much in way of examples :P My instinct is that this would be exchange_dto.go or exchange_transformer.go. Both give context that it is to do with exchanges or loggers, but also highlight that it is there to transform/transfer data from one to another

Similarly, the package being `gct` is quite vague. Yes its interacting with gct, but it is still within gct so its inherit. Perhaps then, calling the package transformer/transfer or something and then you don't need to rename individual files.

I'm still not 100%, but hopefully there's a good name for it so you don't need to add heaps of comments

Reply:
I have not forgotten about this just trying to come up with a better solution that fits in with the Go world !